nemo_skills/pipeline/summarize_results.py

```diff
@@ -61,11 +61,15 @@ def summarize_results(
     remote_tar_dir: str = typer.Option(None, help="Directory where remote tar files are created on clusters"),
     debug: bool = typer.Option(False, help="Print debug information"),
     max_samples: int = typer.Option(-1, help="Limit metric computation only to first `max_samples`"),
+    eval_type: Optional[str] = typer.Option(
```

Comment:
Can you please submit a separate PR with this eval type / map metrics to main? Good to merge it there right away. Maybe also add get_metrics function similar to how it's done for evaluation. And let's always reference metrics through that function, so you'd need to change tests and all init.py for different datasets correspondingly. Please also rename this parameter to "metrics_type"

Reply:
So, instead of importing the metric classes directly, you want always to call get_metrics function and only that function to be imported? So I need to change it everywhere in the code, right?